.changeset/fifty-ghosts-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": minor
+---
+
+The prop `endAdornment` is added to `FinalFormAutocomplete`, it is shown instead of the dropdown chevron
```

Comment:
```suggestion
Add `endAdornment` prop to `FinalFormAutocomplete`

If present, the end adornment will be shown instead of the dropdown chevron.
```